Remotion/Web/UnitTests/Core/ExecutionEngine/UrlMapping/UrlMappingConfigurationTest.cs

```diff
@@ -75,101 +75,112 @@ public void LoadMappingFromFileWithInvalidFilename ()
     [Test]
     public void GetCurrentMapping ()
     {
-      WebConfigurationMock.Current = WebConfigurationFactory.GetExecutionEngineUrlMapping();
-      UrlMappingConfiguration mapping = UrlMappingConfiguration.Current;
-      Assert.That(mapping, Is.Not.Null);
+      using (WebConfigurationFactory.GetExecutionEngineUrlMapping())
```

Comment:
We could also do `using var _ = ...;` instead which would remove the need for indentation